<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddMetaItemWidgetClass</class>
 <widget class="QWidget" name="AddMetaItemWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>273</width>
    <height>67</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="lblIcon">
     <property name="text">
      <string notr="true">lblIcon</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="lneContact"/>
   </item>
   <item row="0" column="2">
    <widget class="CloseButton" name="cbtDelete" native="true"/>
   </item>
   <item row="1" column="1" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="topMargin">
        <number>3</number>
       </property>
       <item>
        <widget class="QLabel" name="lblErrorIcon">
         <property name="text">
          <string notr="true">lblErrorIcon</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="CustomLabel" name="lblError">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string notr="true">lblError</string>
       </property>
       <property name="textFormat">
        <enum>Qt::PlainText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QWidget" name="wdtSelectProfile" native="true"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CustomLabel</class>
   <extends>QLabel</extends>
   <header location="global">utils/customlabel.h</header>
  </customwidget>
  <customwidget>
   <class>CloseButton</class>
   <extends>QWidget</extends>
   <header location="global">utils/closebutton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
